### PR TITLE
feat(profile): allows to update a coordinator address

### LINKED ISSUE #149: account: allow to update coordinator address
This message kind of "transfer" the ownership of a coordinator account

`Coordinator` and `CoordinatorByAddress` entries must be updated

```go
starport message update-coordinator-address ... --module account
```

### PR DESCRIPTION
**Pull request by Pantani**
Closes #149

### What does this PR does?

- Create `update-coordinator-address` structure;
- Implemente `UpdateCoordinatorAddress` method;
- Add `UpdateCoordinatorAddress` test (`TestMsgUpdateCoordinatorAddress`);

### CODE REVIEW

File: proto/profile/tx.proto [hunk 1]
```diff
@@ -21,3 +23,12 @@ message MsgCreateCoordinator {
...
   uint64 coordinatorId = 1;
 }
+
+message MsgUpdateCoordinatorAddress {
+  string address = 1;
+  string newAddress = 2;
+}
+
+message MsgUpdateCoordinatorAddressResponse {
+  uint64 coordinatorId = 1;
```
Comment (lubtd) [inline proto/profile/tx.proto:33]: No need for the coordinator ID in response

File: x/profile/keeper/msg_server_update_coordinator_address.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+	// Check if the new coordinator address is already in the store
+	newCoordAddr, found := k.GetCoordinatorByAddress(ctx, msg.NewAddress)
+	if found {
+		return &types.MsgUpdateCoordinatorAddressResponse{},
+			sdkerrors.Wrap(types.ErrCoordAlreadyExist,
+				fmt.Sprintf("new address already have a coordinator: %d", newCoordAddr.CoordinatorId))
+	}
+
+	coord := k.GetCoordinator(ctx, coordByAddress.CoordinatorId)
```
Comment (lubtd) [inline x/profile/keeper/msg_server_update_coordinator_address.go:32]: Panic if no coordinator (see comment on `MsgUpdateCoordinatorDescription`)

**Review by lubtd — COMMENTED** [review]
👍

**Review by lubtd — APPROVED** [review]
👍

**Review by fadeev — APPROVED** [review]
Looks good!

**Pull request merged**